common/rbnf/de_CH.xml

```diff
@@ -6,424 +6,226 @@ SPDX-License-Identifier: Unicode-3.0
-                <rbnfrule value="0">nullte;</rbnfrule>
-                <rbnfrule value="1">erste;</rbnfrule>
-                <rbnfrule value="2">zweite;</rbnfrule>
-                <rbnfrule value="3">dritte;</rbnfrule>
-                <rbnfrule value="4">vierte;</rbnfrule>
-                <rbnfrule value="5">fünfte;</rbnfrule>
-                <rbnfrule value="6">sechste;</rbnfrule>
-                <rbnfrule value="7">siebte;</rbnfrule>
-                <rbnfrule value="8">achte;</rbnfrule>
-                <rbnfrule value="9">=%spellout-numbering=te;</rbnfrule>
-                <rbnfrule value="20">=%spellout-numbering=ste;</rbnfrule>
-                <rbnfrule value="100">←%spellout-cardinal-masculine←­hundert→%%ste→;</rbnfrule>
-                <rbnfrule value="1000">←%spellout-cardinal-masculine←­tausend→%%ste→;</rbnfrule>
-                <rbnfrule value="1000000">eine Million→%%ste2→;</rbnfrule>
-                <rbnfrule value="2000000">←%spellout-cardinal-feminine← Millionen→%%ste2→;</rbnfrule>
-                <rbnfrule value="1000000000">eine Milliarde→%%ste2→;</rbnfrule>
-                <rbnfrule value="2000000000">←%spellout-cardinal-feminine← Milliarden→%%ste2→;</rbnfrule>
-                <rbnfrule value="1000000000000">eine Billion→%%ste→;</rbnfrule>
-                <rbnfrule value="2000000000000">←%spellout-cardinal-feminine← Billionen→%%ste2→;</rbnfrule>
-                <rbnfrule value="1000000000000000">eine Billiarde→%%ste2→;</rbnfrule>
-                <rbnfrule value="2000000000000000">←%spellout-cardinal-feminine← Billiarden→%%ste2→;</rbnfrule>
-                <rbnfrule value="1000000000000000000">=#,##0=.;</rbnfrule>
-            </ruleset>
-            <ruleset type="spellout-ordinal-n">
-                <rbnfrule value="-x">minus →→;</rbnfrule>
-                <rbnfrule value="x.x">=#,##0.#=;</rbnfrule>
-                <rbnfrule value="0">=%spellout-ordinal=n;</rbnfrule>
-            </ruleset>
-            <ruleset type="spellout-ordinal-r">
-                <rbnfrule value="-x">minus →→;</rbnfrule>
-                <rbnfrule value="x.x">=#,##0.#=;</rbnfrule>
-                <rbnfrule value="0">=%spellout-ordinal=r;</rbnfrule>
-            </ruleset>
-            <ruleset type="spellout-ordinal-s">
-                <rbnfrule value="-x">minus →→;</rbnfrule>
-                <rbnfrule value="x.x">=#,##0.#=;</rbnfrule>
-                <rbnfrule value="0">=%spellout-ordinal=s;</rbnfrule>
-            </ruleset>
-            <ruleset type="spellout-ordinal-m">
-                <rbnfrule value="-x">minus →→;</rbnfrule>
-                <rbnfrule value="x.x">=#,##0.#=;</rbnfrule>
-                <rbnfrule value="0">=%spellout-ordinal=m;</rbnfrule>
-            </ruleset>
-        </rulesetGrouping>
-    </rbnf>
+	<identity>
+		<version number="$Revision$"/>
+		<language type="de"/>
+		<territory type="CH"/>
+	</identity>
+	<rbnf>
+		<rulesetGrouping type="SpelloutRules">
```

Comment:
I think we need to modify the generation in order to work better with RBNF rules, with the changes made in this release. Doesn't have to be now, but we shouldn't go out with this.
That is, we need to preserve line breaks inside of the <rbnfRules> elements.

Reply:
it's also missing the CDATA